app/main.js

```diff
@@ -95,7 +95,9 @@ app.on('ready', function () {
   });
 
   ipc.on('badge', function(event, arg) {
-    app.dock.setBadge(arg.badge_text);
+    if (process.platform === 'darwin') {
+      app.dock.setBadge(arg.badge_text);
+    }
```

Comment:
You can also replace these with something that 'linux should do', if there is such a thing. It's been a long time since I did linux on the desktop, though.